store/src/java/com/zimbra/cs/mailbox/calendar/Invite.java

```diff
@@ -579,6 +641,9 @@ public static Invite decodeMetadata(int mailboxId, Metadata meta, CalendarItem c
         String name = meta.get(FN_NAME, "");
         String loc = meta.get(FN_LOCATION, null);
 
+        String opType = meta.get(FN_OPERATION_TYPE,null);
+        String tagCol = meta.get(FN_TAG_COLOUR,null);
+
```

Comment:
extra space, check in other part of code changes as well

Reply:
Sure I have modified it.